cross/busybox/Makefile

```diff
@@ -27,7 +27,7 @@ busybox_configure:
 
 .PHONY: busybox_compile
 busybox_compile:
-	$(RUN) CROSS_COMPILE=$(TC_PATH)$(TC_PREFIX) $(MAKE) busybox
+	$(RUN) CROSS_COMPILE=$(TC_PATH)$(TC_PREFIX) $(MAKE) -j $(NCPUS) busybox
```

Comment:
Are we back to where we need to specify which custom compile target is build in parallel? Kind of like #4783 did?` it's easy to forget, so I liked the default and maybe have an easy opt out. At the same time the errors are not obvious when they are coursed by a parallel build, so maybe this is the best thing after all. CC/ @hgy59

Reply:
Not at all!  `busybox` overrides default `COMPILE_TARGET` and as such doesn't honor the `MAKEFLAGS` as spawn-up with `-@MAKEFLAGS= $(MAKE)  ...`

Although, there may be a way for theses exceptions to force the `MAKEFLAGS` ...  I'll see if I can come up with something.

Reply:
@publicarray I've looked at it and no, I haven't found any way to play with `MAKEFLAGS` for theses specific exceptions.  A such, packages exceptionally overriding the `COMPILE_TARGET` must add `-j $(NCPUS)` to parallel build within their `custom_compile_target`. 

For the `busybox` case, it was really easy to migrate to using default `COMPILE_TARGET` and thus parallel build by default (see https://github.com/SynoCommunity/spksrc/pull/4854/commits/e2935b623cfe7a115e8b8c859707d0e963357ec3)